src/components/OverlayTooltip/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { OverlayTrigger, Tooltip } from 'patternfly-react'
+
+class OverlayTooltip extends React.Component {
+  // constructor (props) {
+  //   super(props)
+  //   var id = props.id
+  //   var tooltip = props.tooltip
+  //   var placement = props.placement
+  //   var children = props.children
+  // }
+  render () {
+    return (
+      <OverlayTrigger
+        overlay={
+          <Tooltip id={this.props.id}>{this.props.tooltip}</Tooltip>
+        }
+        placement={this.props.placement}
+      >
+        { this.props.children }
+      </OverlayTrigger>
+    )
+  }
+}
+OverlayTooltip.propTypes = {
+  id: PropTypes.string.isRequired,
+  tooltip: PropTypes.string.isRequired,
+  placement: PropTypes.string.isRequired,
+  children: PropTypes.node,
```

Comment:
Why children is not required? Because I don't see any case of empty `children` in the code.